org.envirocar.app/res/layout/activity_signin.xml

```diff
@@ -55,7 +55,7 @@
             android:layout_height="wrap_content"
             android:text="@string/sign_in"
             android:textColor="@color/white_cario"
-            android:textSize="26dp"
+            android:textSize="26sp"
```

Comment:
Donot use sp , Here the Textview has specified box/area, To be on saferside we use dp.